.travis.yml

```diff
@@ -1,38 +1,52 @@
 language: c
-script: make && make test # no ./configure
+script: meson build && cd ./build && ninja
 
 # Uncomment me if this gets annoying
 #
-# notifications: 
-#   email: false 
+# notifications:
+#   email: false
+
+before_install:
+  - wget https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip
+  - unzip ninja-linux.zip
+  - sudo mv ninja /usr/bin/
+  - wget https://github.com/libuv/libuv/archive/v1.19.2.tar.gz
```

Comment:
Is the correct version of `libuv` neither a submodule nor available as a package on LTS? I'm leery of installing things informally like this, for the CI it's probably fine but I expect it translates to everyone trying to build urbit also having to resolve these dependencies.